src/consensus/params.h

```diff
@@ -11,6 +11,15 @@
 
 namespace Consensus {
 
+enum BuriedDeployment
+{
+    // buried deployments get negative values to avoid overlap with SignalledDeployment
+    DEPLOYMENT_CLTV = -255,
+    DEPLOYMENT_DERSIG,
+    DEPLOYMENT_CSV,
+    DEPLOYMENT_SEGWIT,
```

Comment:
prefer explicitly numbering these.

-255 is an extremely odd number, can you clarify why you picked it? It doesn't fit in a signed byte.

Reply:
Explicitly numbering them risks accidentally repeating a number. No particular objection to a different starting point. Could perhaps make it `enum DeploymentPos : uint8_t` and `enum BuriedDeployment : int16_t { DEPLOYMENT_CLTV = 256, // larger than largest possible DeploymentPos value`.

(I guess `2**n` is an "extremely even" number, so calling  `+/- 2**n-1` "extremely odd" makes a lot of sense...)